docs/quick-start/write-data.txt

```diff
@@ -0,0 +1,97 @@
+.. _laravel-quick-start-write-data:
+
+=================================
+Post an API Request to Write Data
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Function to Store Data
+
+      Add logic to save the data from a request in the ``MovieController.php``
+      file in the ``app/Http/Controllers/`` directory. Replace the existing
+      placeholder ``store()`` method with the following code:
+
+      .. code-block:: php
+
+         public function store(Request $request)
+         {
+             $data = $request->all();
+             $movie = new Movie();
+             $movie->fill($data);
+             $movie->save();
+         }
+
+   .. step:: Add a Route for the Controller Function
+
+      Add the API route to map to the ``store()`` method in the
+      ``routes/api.php`` file.
+
+      .. code-block:: php
+
+         use App\Http\Controllers\MovieController;
+
+         // ...
+
+         Route::resource('movies', MovieController::class)->only([
+           'store'
+         ]);
+
+
+   .. step:: Update the Movie Model
+
+      Add the movie detail fields to the ``$fillable`` array
+      of the ``Movie`` model. After adding them, the ``Movie``
+      class at ``app/Models/Movie.php`` should contain the
+      following code:
+
+      .. code-block:: php
+         :emphasize-lines: 4
+
+         class Movie extends Model
+         {
+            protected $connection = 'mongodb';
+            protected $fillable = ['title', 'year', 'runtime', 'imdb', 'plot'];
+         }
+
+   .. step:: Post a Request to the API
+
+      Create a file called ``movie.json`` and insert the following data:
```

Comment:
Q: should this code example include how to insert the data, rather than just the data itself, or are you assuming the user already knows how to insert?

Reply:
The insert is defined in "Create the Function to Store Data". I will change the title to "Create the Function to Insert Data" to make it more apparent for anyone that might be specifically looking for that word.
This step is chronologically after that one and describes how to perform a POST request with sample data so that the application calls that insert code.